ossrelease/github_release.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
+import argparse
+
+# import ossrelease modules
+import conf
+
+def parse_args():
+    '''
+    Parse the CLI options.
+    '''
+    # Define parser and set up basic options
+    parser = argparse.ArgumentParser(description='Create a release on github.')
+    parser.add_argument('--salt-ver', help='Specify salt version to create release')
+
+    return parser.parse_args()
+
+def main():
+    args = parse_args()
+    opts = conf.get_conf()
+
+    repo = Github(opts['GITHUB_TOKEN']).get_repo('saltstack/salt')
+
+    # create release
+    # todo: make sure the tag we are about to create a release for already exists
```

Comment:
I know this is a first pass, but I'd like to see this kind of validation happen before this script is merged in.

Reply:
k i'll add the [WIP] for now until i can get to this again. might be a while.